doc/conf.py

```diff
@@ -41,16 +41,28 @@
 
 # General information about the project.
 project = u'screed'
-copyright = u'2012, Alex Nolley and Titus Brown'
+copyright = u'2015, Alex Nolley and Titus Brown'
```

Comment:
On Tue, Feb 24, 2015 at 02:01:13PM -0800, Michael R. Crusoe wrote:
> > @@ -41,16 +41,28 @@
> >  
> >  # General information about the project.
> >  project = u'screed'
> > -copyright = u'2012, Alex Nolley and Titus Brown'
> > +copyright = u'2015, Alex Nolley and Titus Brown'
> 
> 2012-2015

copyright should be to Michigan State University, too!